isis/src/messenger/apps/mdis2isis/tsts/default/Makefile

```diff
@@ -10,9 +10,9 @@ commands:
 	  to=$(OUTPUT)/mdis2isisTruth.pvl> /dev/null
 	$(APPNAME) from=$(INPUT)/EN0089569526M.IMG \
 	  to=$(OUTPUT)/mdis2isisLutTruth.cub > /dev/null
-	catlab from=$(OUTPUT)/mdis2isisTruth.cub \
+	catlab from=$(OUTPUT)/mdis2isisLutTruth.cub \
```

Comment:
Why Lut?

Reply:
I changed it to Lut because if we used mdis2isisTruth.cub, we would be checking the label for mdis2isisTruth.cub again, since the first cat call does this, instead of the LutTruth cube we just created.

Reply:
What does Lut stand for though?

Reply:
I am not sure. The pvl files were named this way before I made my changes. I renamed the cubes being passed into catlab to make sure it was testing the correct cube.

Reply:
Is there a reasonable replacement for "Lut" that makes more sense in the context of this test? If not that is okay.

Reply:
Hi Makayla, nothing really stood out to me in the pvl file that would give me a new name.